api/src/controllers/collection-controller.js

```diff
@@ -64,8 +61,26 @@ exports.protectedPut = async function (args, res, next) {
     defaultLog.info(`protectedPut - you must provide an object`);
     queryActions.sendResponse(res, 400, {});
     next();
+  }  
+
+  let obj = null;
+  try {
+    // obj = await Put.updateById(collectionId, updateObj);
```

Comment:
remove this commented line

Reply:
Fixed in https://github.com/bcgov/NRPTI/pull/690/commits/2bd4d09487e3927447fa6e30c778887452e5a60b